app/src/main/java/pm/gnosis/heimdall/data/repositories/models/Safe.kt

```diff
@@ -4,6 +4,8 @@ import java.math.BigInteger
 
 sealed class AbstractSafe
 
-data class Safe(val address: BigInteger, val name: String? = null): AbstractSafe()
+data class Safe(val address: BigInteger, val name: String? = null) : AbstractSafe()
 
-data class PendingSafe(val hash: BigInteger, val name: String? = null): AbstractSafe()
+data class PendingSafe(val hash: BigInteger, val name: String? = null) : AbstractSafe()
+
+data class SafeWithInfo(val safe: Safe, val info: SafeInfo? = null) : AbstractSafe()
```

Comment:
It feels weird adding this as a repository model, but it is never used by the repository